tools/webhook-ingestion-pipeline/webhook_endpoint/main.py

```diff
@@ -51,6 +55,22 @@ def handle_invalid_usage(error):
     response.status_code = error.status_code
     return response
 
+
+def _append_metadata(json_data):
```

Comment:
nit: this is not 'append' really it's falling back to default metadata.
perhaps use `_set_missing_metadata`